packages/cli/package.json

```diff
@@ -18,7 +18,7 @@
     "lint:fix": "eslint 'src/**/*.js' 'tests/**/*.js' --fix",
     "pre-commit": "lint-staged",
     "test": "jest --filter=./tests/testFilter.js",
-    "jest": "jest --filter=./tests/testFilter.js",
```

Comment:
Providing a `jest` script prevents us from running `yarn jest` without the args specified here.